evms/gen_oplog.go

```diff
@@ -75,8 +83,14 @@ func (o *opLog) UnmarshalJSON(input []byte) error {
 	if dec.Pc != nil {
 		o.Pc = *dec.Pc
 	}
+	// Depends on Geth EOF support
```

Comment:
Does it? We can enable this, I think, as long as we don't use it for diffing later on

Reply:
This one is fine, it's from goevmlab code.